src/include/Plugin.hpp

```diff
@@ -41,7 +41,7 @@ class DevicePlugin {
 	virtual ~DevicePlugin() {}
 
 	// Helper for loading all DevicePlugin's
-	static std::optional<std::vector<boost::shared_ptr<DevicePlugin>>> loadPlugins();
```

Comment:
Was there any reason for using `boost` here when the std provides this that I'm not aware of?

Reply:
It's because we load the plugins with Boost's API, and it returns `boost::shared_pointer` here:
https://github.com/Lurkki14/tuxclocker/blob/bfff550b9d0167dc05574e33a04a68b16b20edb0/src/lib/Plugin.cpp#L27-L28

Unless we change the way plugins are loaded, I don't think there's much value to change it to `std::shared_ptr`.

This also introduces a compile error due to these mismatching types.

Reply:
Makes sense! Thanks!
I didn't hit any compiler issues when using the readme steps, or using CLion to build the `all` target, so was not aware of this.